src/typescript/sdk/src/index.ts

```diff
@@ -1 +1,26 @@
-export type { Side } from "../types/global";
+import { type AptosAccount, AptosClient, type MaybeHexString } from "aptos";
+
+import { EconiaTransactionBuilder } from "./tx-builder";
+
+export * as events from "../types/events";
+export * as order from "../types/order";
+
+export class EconiaClient {
+  public readonly econiaAddress: MaybeHexString;
+  public readonly aptosClient: AptosClient;
+  public readonly userAccount: AptosAccount;
+
+  public constructor(
+    nodeUrl: string,
+    econiaAddress: MaybeHexString,
+    userAccount: AptosAccount
+  ) {
+    this.econiaAddress = econiaAddress;
+    this.aptosClient = new AptosClient(nodeUrl);
+    this.userAccount = userAccount;
+  }
+
+  public createTx(): EconiaTransactionBuilder {
+    return new EconiaTransactionBuilder(this);
+  }
```

Comment:
I find this design puzzling. Having a function called `createTx` return an `EconiaTransactionBuilder` seems unintuitive.